package.json

```
@@ -93,16 +93,16 @@
     "npm-run-all2": "^6.2.6",
     "prettier": "^3.5.2",
     "prettier-plugin-packagejson": "^2.5.9",
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0",
+    "react": "19.2.1",
+    "react-dom": "19.2.1",
     "react-is": "^19.0.0",
     "sanity": "^3.99.0",
     "semantic-release": "^24.2.3",
     "styled-components": "^6.1.15",
     "typescript": "5.9.3"
   },
   "peerDependencies": {
-    "react": "^18.3 || ^19",
+    "react": "19.2.1",
```

We can't remove the `^18.3` allowance, that would be a breaking change, and we also can't use exact versions in `peerDependencies`
```suggestion
    "react": "^18.3 | ^19.2.1",
```